extensions/amp-script/0.1/user-activation-tracker.js

```diff
@@ -100,13 +100,13 @@ export class UserActivationTracker {
       return;
     }
     this.inLongTask_ = true;
```

Comment:
Is it worth checking if this is already `true`? Should we be able to assert on it?

Reply:
What does that buy us?

Reply:
Based on this from the PR description:
We need it to be a sequence of START --> END --> START --> END.

Would it be possible or problematic if we had START --> START --> END --> END?

Reply:
Yes! That is the exact problem scenario. We could do a devAssert as a form of documentation for ourselves?

Reply:
SG, let's do that!

Reply:
Updated with the `devAssert`